burgle.lic

```diff
@@ -13,77 +13,166 @@ class Burgle
   include DRCT
   include DRCI
 
-
   def initialize
 
     arg_definitions = [
       [
-        { name: 'start', regex: /start/i, description: 'Required: prevents accidentally running burgle and protects against burgle typo for starting script' },
+        { name: 'start', regex: /start/,optional: false, description: 'Required: prevents accidentally running burgle and protects against burgle typo for starting script' },
+        { name: 'entry', options: %w[lockpick rope cycle], optional: true, description: 'Override yaml setting for entry_type.' },
+        { name: 'roomid', regex: /\d+/, optional: true, description: 'Override yaml setting and go to room id (#) specified.  The room shold be chosen VERY carefully.'},
+        { name: 'loot_type', options: %w[drop keep], optional: true, description: 'Override yaml setting for loot. (items on item_whitelist are always kept.)' },
       ]
     ]
 
     args = parse_args(arg_definitions)
 
+    @entry_type = args.entry if args.entry
+    @burgle_room = args.roomid if args.roomid
+    @loot_type = args.loot_type if args.loot_type
+
+    #hash which maps directions to their reverse, for backing out of the house
+    @reverse_direction_map = {
```

Comment:
💡 We do the same thing in https://github.com/rpherbig/dr-scripts/blob/4fcbc37f7eb85f733643ea43b102adb00c2bc1cc/dusk-labyrinth.lic#L22

We're not at the point I'm insisting on commonizing this, but it's something to keep an eye on.